src/components/Sprint.vue

```diff
@@ -67,10 +72,29 @@
         const sprintId = this.sprint.id;
         
         FirebaseService.removeSprint(projectId, sprintId);
-        store.commit('removeSprintFromProject', {
+        this.$store.commit('removeSprintFromProject', {
           projectId,
           sprintId
         });
+      },
+      setUpdateModel() {
+        this.updatedSprint = {
+          startDate: this.sprint.startDate,
+          endDate: this.sprint.endDate
+        }
+        return this.editMode = true;
```

Comment:
What is the purpose of returning this?

Reply:
Because it had a hissy fit when I didn't. I will double check that it was because of this, if not I will remove it as it wasn't what I initially did.

Reply:
Just checked, it now works, I have removed it.